.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: test
+on:
+  push:
```

Comment:
Would you agree that it is not necessary to run the tests for every Push action, but only as a required check before merging?

Reply:
I was actually looking to see if anyone had already done something like this for PRs. Would be most handy. I agree checking every push is unnecessary right now, and just making sure tests pass for each PR would be ideal.

Reply:
GitHub's Python docs seem to favor `on: [push]` but I am flexible.  It is useful if contributors and reviewers can see the tests pass before starting the code review.
* https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#using-multiple-python-versions

Reply:
Thanks @cclauss. I am not very familiar with setting up github checks, so do I understand correctly that having them on push would re-trigger the checks on every push? In that case I agree having them on push makes sense.

Reply:
Simplified to just `on: [push]`.

Reply:
The nice explanation can be found at
* FactoryBoy/factory_boy#1075